apps/web/components/eventtype/EventTeamTab.tsx

```diff
@@ -255,20 +343,47 @@ const Hosts = ({
     });
   }, [schedulingType, resetField, getValues, submitCount]);
 
+  const formMethods = useFormContext<FormValues>();
+  const [assignAllTeamMembers, setAssignAllTeamMembers] = useState(
+    formMethods.getValues("assignAllTeamMembers") ?? false
+  );
+
   return (
     <Controller<FormValues>
       name="hosts"
       render={({ field: { onChange, value } }) => {
         const schedulingTypeRender = {
           COLLECTIVE: (
-            <CheckedHostField
-              value={value}
-              onChange={onChange}
-              isFixed={true}
-              options={teamMembers.sort(sortByLabel)}
-              placeholder={t("add_attendees")}
-              labelText={t("team")}
-            />
+            <div className="border-subtle mt-6 space-y-5 rounded-lg border px-4 py-6 sm:px-6">
+              <div className="flex flex-col gap-4">
+                <AssignAllTeamMembers
+                  assignAllTeamMembers={assignAllTeamMembers}
+                  setAssignAllTeamMembers={setAssignAllTeamMembers}
+                  onActive={() =>
+                    formMethods.setValue(
+                      "hosts",
+                      teamMembers.map((teamMember) => ({
+                        isFixed: true,
+                        userId: parseInt(teamMember.value, 10),
+                      }))
+                    )
+                  }
+                  onInactive={() => formMethods.setValue("hosts", [])}
+                />
+                {assignAllTeamMembers ? (
+                  <></>
```

Comment:
```suggestion
                  <div className="mb-4"></div>

```

Before: 
<img width="948" alt="Screenshot 2024-01-24 at 10 44 09 AM" src="https://github.com/calcom/cal.com/assets/30310907/8be282b2-328b-425c-8adf-490244cadcf9">

After: 
<img width="978" alt="Screenshot 2024-01-24 at 10 43 39 AM" src="https://github.com/calcom/cal.com/assets/30310907/29f9d880-ada5-47d7-a655-706253d6073f">